lib/dialyzer/test/behaviour_SUITE_data/results/supervisor_incorrect_return

```diff
@@ -1,2 +1,2 @@
 
```

Comment:
```suggestion
%% %CopyrightBegin%
%%
%% SPDX-License-Identifier: Apache-2.0
%%
%%
%% %CopyrightEnd%
```

To get rid of the licenase header warning.

Reply:
Thank you!